go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/restic/restic
 
+replace github.com/minio/minio-go/v7 => ../minio-go
```

Comment:
That references a local folder on your disk. Why not directly reference the commits from the mentioned minio-go PR?

Reply:
Hey, I was not aware of that method initially. 😅

I will update this to the specific commit hash. 

I have marked the MR in draft for the same.